doc/distrib/NodeHelpFiles/_review/Dynamo.Nodes.SelectEdges.md

```diff
@@ -0,0 +1,10 @@
+## In Depth
+
+`Select Edges` displays a cursor in Revit that allows you to select multiple edges. The objects returned are curves or lines that represent the selected edges.
+
+In the example below, two lines are returned that represent the top edges of a parapet in the current document (file).
+
```

Comment:
For clarity and keeping it consistent, suggest adding: Additionally, Select Edges reports the owning element ids of the edges that are selected.